pkgs/applications/networking/instant-messengers/fluffychat/default.nix

```
@@ -1,35 +1,24 @@
 { lib
-, stdenv
-, fetchzip
+, fetchFromGitLab
 , imagemagick
-, autoPatchelfHook
-, gtk3
-, libsecret
-, jsoncpp
-, wrapGAppsHook
+, flutter
 , makeDesktopItem
-, openssl
-, olm
 }:
 
-let
+flutter.buildFlutterApplication rec {
   version = "1.10.0";
```

v1.11.1 is released:

```suggestion
  version = "1.11.1";
```

---

1.11.1 is not yet released, but 1.11.0 is. I have upgraded to that.